teams/brno.yml

```
@@ -523,21 +523,21 @@
 #   links:
 #   - mail-link: mailto:iveta.horrova@cz.ibm.com
 
-- name: Edit Lelkes
-  img: img/brno/team/blank.png
-  role: Koučka
-  links:
-  - mail-link: mailto:balkoedi@gmail.com 
+#- name: Edit Lelkes
+#  img: img/brno/team/blank.png
+#  role: Koučka
+#  links:
+#  - mail-link: mailto:balkoedi@gmail.com 
 
 # - name: Jaroslav Šafka
 #   img: img/brno/team/jareks.jpg
 #   role: Kouč
 #   links:
 #   - mail-link: mailto:pyladies zavináč safka.org
 
-# - name: Katka Chmelařová
-#   img: img/brno/team/blank.png
-#   role: Kouč
+ - name: Katka Chmelařová
+   img: img/brno/team/blank.png
+   role: Kouč
```

```suggestion
  img: img/brno/team/blank.png
  role: Kouč
```
Smazání jedné mezery na začátku.